packages/oauth/src/index.ts

```diff
@@ -782,3 +699,4 @@ interface AuthTestResult extends WebAPICallResult {
 }
 
 export { Logger, LogLevel } from './logger';
+export { MemoryInstallationStore, FileInstallationStore } from './stores';
```

Comment:
Both now exported for use directly in projects so that the developer might import that which they prefer to work with.

Reply:
How about doing this instead? No need to modify when we add more builtin store implementations.
```typescript
export * from './stores';
```